app/views/ViewUtils.scala

```diff
@@ -37,13 +37,43 @@ object ViewUtils {
     if (form.hasErrors || form.hasGlobalErrors) messages("error.title.prefix") else ""
   }
 
+  def formatUkAddress(ukAddress: UkAddress): String =
```

Comment:
Probably this class will change a lot after you pass an address object to the template

Reply:
We now have three Address classes: Address, InternationalAddress, UkAddress. No need for that. We should merge them.

Reply:
Please raise anothger ticket for that as it is nothing to do with this.